### ISSUE TITLE
Include affinities in increase/decrease queries?

### ISSUE THREAD

**Issue by cbizon** [NCATSTranslator/Feedback#820]
When we ask chemical increases/decreases gene type questions, we get a lot of lookup results, which is good.  Many of these are coming from places like binding db.  Many of these should include things like pIC50s.  This kind of information would be incredibly valuable and should be
1) returned
2) displayed

**Comment by cbizon** [comment]
e.g. PK: a5731faf-7cd8-42f2-81ef-5cf6030931fa

For example: According to bindingdb (our primary source) ruxolitinib has a 3 nM IC50, which seems worth including even at the table level.

**Comment by dnsmith124** [comment]
@Genomewide thoughts on the above?

**Comment by gprice1129** [comment]
The UI team needs to discuss internally before assigned to a release.

**Comment by Genomewide** [comment]
I like the idea. And we should display that if returned in the trapi at least as part of the EPC.

**Comment by Genomewide** [comment]
@cbizon is that information returned in the attributes?

**Comment by cbizon** [comment]
Yes, although it's somewhat hit or miss depending on which source things are coming from:
![image](https://github.com/user-attachments/assets/24ad8c8b-7e5c-4d69-97f6-98afc32d9bf0)

**Comment by Genomewide** [comment]
ok. we should look at adding this to EPC or other place. If it is edge related info, I think it should go in the EPC myself. If we put it there for some intermediate edges and not for the inferred or direct edge (putting at the table level) that could be confusing? But that is without giving it much thought yet.

**Comment by Genomewide** [comment]
@cbizon  I tried to find this in the ARAX interface and the raw json and can not find this info. Was your screenshot from the ARAX, Unsecret, or ARS data? I can not find the pKi anywhere. I also searched the combined json from the ARS and don't see it or the 'affinity_parameter' either. Please help.
